tests/foreman/ui/test_activationkey.py

```diff
@@ -795,7 +793,68 @@ def test_positive_update_activation_key_4(self, cv2_name):
                                  (common_locators["alert.success"]))
             selected_cv = self.activationkey.get_attribute(name, cv_locator)
             self.assertEqual(cv2_name, selected_cv)
-            # TODO: Need to check for RH Product too
+
+    @run_only_on('sat')
+    @data(*valid_data_list())
+    def test_positive_update_activation_key_with_rh_product(self, cv2_name):
+        """@Test: Update Content View in an Activation key
+
+        @Feature: Activation key - Positive Update
+
+        @Steps:
+        1. Create an activation key
```

Comment:
Bah. This is still syntactically incorrect. Next time, please read my example replacement text more carefully or test things out with a reST compiler like `rst2html`, `make docs` or [this website](http://rst.ninjs.org/).

Not a blocker. The rest of this PR looks good. Let's get it merged.